### PR TITLE
Template renderer first steps

### PR DESCRIPTION
**Pull request by juandspy**
# Description

Added some basic tests for the template renderer.

## Type of change

- New test feature file

## Testing steps

Locally.

## Checklist
* [ ] Pylint passes for Python sources
* [ ] sources has been pre-processed by Black
* [ ] updated documentation wherever necessary
* [ ] new tests can be executed both locally and within docker container

### CODE REVIEW

File: insights_content_template_renderer_tests.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+function prepare_venv() {
+    echo "Preparing environment"
+    # shellcheck disable=SC1091
+    virtualenv -p python3 venv && source venv/bin/activate
+    echo "Environment ready"
+}
+
+function install_service() {
+    cd "$PATH_TO_LOCAL_TEMPLATE_RENDERER"
```
Comment (epapbak) [inline insights_content_template_renderer_tests.sh:38]: you should clone it if not there. something like https://github.com/RedHatInsights/insights-behavioral-spec/blob/438bbe1ba59f04e18b76aabebb30238b1ecac2f9/insights_content_service_test.sh#L54

File: requirements/insights_content_template_renderer.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+jsonschema
+minio
+kafka-python
```
Comment (epapbak) [inline requirements/insights_content_template_renderer.txt:3]: why?

File: features/steps/common_http.py [hunk 3]
```diff
@@ -160,7 +160,9 @@ def set_rest_api_prefix(context, prefix):
 @when("I access endpoint {endpoint} using HTTP GET method")
 def access_rest_api_endpoint_get(context, endpoint):
     """Send GET HTTP request to tested service."""
-    url = f"http://{context.hostname}:{context.port}/{context.api_prefix}/{endpoint}"
+    base = f"http://{context.hostname}:{context.port}"
+    path = f"{context.api_prefix}/{endpoint}".replace("//", "/")
```
Comment (tisnik) [inline features/steps/common_http.py:164]: replaceall?

File: requirements/insights_content_template_renderer.txt (hunk 2 above)
Comment (tisnik) [reply requirements/insights_content_template_renderer.txt:3]: inded

**Review by tisnik — CHANGES_REQUESTED** [review]

File: features/steps/common_http.py (hunk 3 above)
Comment (juandspy) [reply features/steps/common_http.py:164]: WDYM? I'm replacing all the double slash in the subdirectory, not the ones in `http://`

Comment (juandspy) [reply features/steps/common_http.py:164]: the purpose is to avoid things like `http://localhost:8000//api/v1//rendered_reports`

Comment (tisnik) [reply features/steps/common_http.py:164]: sure, but those `//` might be used multiple times (and not - I'm not talking about `base` part)

**Review by tisnik — APPROVED** [review]
ok

**Review by Bee-lee — APPROVED** [review]

**Review by epapbak — APPROVED** [review]
thanks!

**Pull request merged**